lib/inferno/dsl/fhir_evaluation/rules/abstract_has_examples_rule.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
```

Comment:
This abstract rule isn't used by either of the two being added here, so let's leave this out for now

Reply:
Remove for now